client/app/services/custom-button/_custom-button.sass

```diff
@@ -19,3 +19,14 @@
       border-color: $app-color-light-blue-4
       color: $app-color-medium-gray-2
       text-decoration: none
+.custom-button-action
+  &.disabled
+    color: $color-gray-3
```

Comment:
but `$color-gray-3` doesn't exist in `_app_colors.sass` ?  (below its added to `_colors.sass`

Reply:
Do you think it should be in app_colors instead of colors?

Reply:
So `_colors.sass` isn't imported in this file, `$color-gray-3`  doesn't exist in `_app_colors.sass` ya shouldn't be seeing any color change in the case of an absent variable, so I'm think we're either pulling color from another disabled class (which removing the sass work would reveal) or we have a bigger issue with sass variable bleed